From 883cd2fc3853b936a341287b7f62e7355012c1b1 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Thu, 19 Nov 2015 16:43:18 -0800
Subject: [PATCH 4/5] Use trimmed down glibc C locale

We don't do any translations with pacrunner, so we can avoid using the UTF8
locale and potentially save some space/memory.

Signed-off-by: Tudor Marcu <tudor.marcu@intel.com>
---
 src/main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/main.c b/src/main.c
index f3767e8..6a503b7 100644
--- a/src/main.c
+++ b/src/main.c
@@ -154,6 +154,9 @@ int main(int argc, char *argv[])
 	DBusError err;
 	guint signal;
 
+	if (setenv("LC_ALL", "C", 1))
+		perror("Cannot set LC_ALL to C");
+
 	context = g_option_context_new(NULL);
 	g_option_context_add_main_entries(context, options, NULL);
 
-- 
2.16.2

